controls/cis_eks.yml

```diff
@@ -0,0 +1,647 @@
+
+          - The worker node guest operating system (including updates and security patches)
+              - Amazon EKS follows the shared responsibility model for CVEs and security patches on managed node groups. Because managed nodes run the Amazon
+                EKS-optimized AMIs, Amazon EKS is responsible for building patched versions of these AMIs when bugs or issues are reported and we are able to publish a fix. However, customers are responsible for deploying these patched AMI versions to your managed node groups.
+
+        - Other associated application software:
+          - Setting up and managing network controls, such as firewall rules
+          - Managing platform-level identity and access management, either with or in addition to IAM
+
+        - The sensitivity of your data, your company’s requirements, and applicable laws and regulations
+
+    AWS is responsible for securing the control plane, though you might be able to configure certain options based on your requirements.
+  notes: |-
+    As noted, evaluating the security of the control plane is not the
+    responsibility of customers or deployers, but of Amazon instead. Thus, this
+    profile won't cover this.
+  status: not applicable
+  rules: []
+
+- id: '2'
+  title: '2 Control Plane Configuration'
+  description: |-
+    This section contains recommendations for Amazon EKS control plane logging
+    configuration. Customers are able to configure logging for control plane in
+    Amazon EKS.
+  levels:
+  - level_2
+  status: not applicable
+  controls:
+  - id: '2.1'
+    title: '2.1 Logging'
+    status: not applicable
+    levels:
+    - level_2
+    controls:
+    - id: 2.1.1
+      title: '2.1.1 Enable audit Logs (Manual)'
+      description: |-
+        Description:
+
+        The audit logs are part of the EKS managed Kubernetes control plane logs that are managed
+        by Amazon EKS. Amazon EKS is integrated with AWS CloudTrail, a service that provides a
+        record of actions taken by a user, role, or an AWS service in Amazon EKS. CloudTrail
+        captures all API calls for Amazon EKS as events. The calls captured include calls from the
+        Amazon EKS console and code calls to the Amazon EKS API operations.
+
+        Rationale:
+
+        Exporting logs and metrics to a dedicated, persistent datastore such as CloudTrail ensures
+        availability of audit data following a cluster security event, and provides a central location
+        for analysis of log and metric data collated from multiple sources.
+      # TODO: Evaluate if it's feasible to do an automated check for this
+      # control. This might require that the Compliance Operator acquire the
+      # ability to probe AWS endpoints (like the EKS one) to be able to do such
+      # checks.
+      notes: |-
+        It's not feasible to do such a check via automated means. e.g. via the
+        Compliance Operator using OpenSCAP. This has to be checked manually.
```

Comment:
I'm not sure if we can rely on the `aws` command line, but I was able to fetch this information using `aws eks describe-cluster $CLUSTER_NAME`
```
CLUSTERLOGGING  False
TYPES   api
TYPES   audit
TYPES   authenticator
TYPES   controllerManager
TYPES   scheduler
```

Manually enabling logging through the control plane gives me the following results afterwards:

```
CLUSTERLOGGING  True
TYPES   api
TYPES   audit
TYPES   authenticator
TYPES   controllerManager
TYPES   scheduler
```

Reply:
Right, but either way, we don't have the means to call `aws eks` unless we enhance the compliance operator. So, we need to evaluate the feasibility of this.

Reply:
Ack - sounds like we'll need something in CO to do that, then?

Reply:
Yep, let's discuss this one outside of this PR